charts/gha-runner-scale-set-controller/values.yaml

```diff
@@ -83,3 +83,15 @@ flags:
   ## Restricts the controller to only watch resources in the desired namespace.
   ## Defaults to watch all namespaces when unset.
   # watchSingleNamespace: ""
+
+# If metrics object is not provided, following flags won't be applied:
+# --metrics-addr, --listener-metrics-addr, --listener-metrics-endpoint
+# Keep in mind, metrics are exposed by default (flags have default values)
+# specifying the default port and default endpoint.
+#
+# To turn off metrics, specify empty strings for controllerAddr and listenerAddr
+metrics:
+  controllerAddr: ":8080"
+  listenerAddr: ":8080"
```

Comment:
Do we want to use the fully qualified name?
Ex:
- `controllerManagerAddr`
- `autoscalinglistenerAddr`